app/_components/FileUpload/FileUpload.jsx

```diff
@@ -31,7 +31,7 @@ const FileUpload = ({buttonText}) => {
     try {
       for (let i = 0; i < selectedFiles.length; i++) {
         const file = selectedFiles[i];
-        const bucketName = 'datajam-expense-parser'; // Replace with your S3 bucket name
+        const bucketName = 'eclassifierbucket'; // Replace with your S3 bucket name
```

Comment:
revert this file. You should merge the dev. We no longer need the bucketName in this. Everything is handled at the AWSClient.js